3rd-task/index.html

```diff
@@ -0,0 +1,374 @@
+                    <p>It is a long establ ished fact that a reader will be distracted by the readable content.</p>
+                </li>
+                <li>
+                    <span>
+                        <img src="assets/spurt.png" alt="ontime"/>
+                    </span>
+                    <h5>Full Support</h5>
+                    <p>It is a long establ ished fact that a reader will be distracted by the readable content.</p>
+                </li>
+                <li>
+                    <span>
+                        <img src="assets/custome.png" alt="ontime"/>
+                    </span>
+                    <h5>Easy to Customize</h5>
+                    <p>It is a long establ ished fact that a reader will be distracted by the readable content.</p>
+                </li>
+                <li>
+                    <span>
+                        <img src="assets/mobile.png" alt="ontime"/>
+                    </span>
+                    <h5>Ontime Completed</h5>
+                    <p>It is a long establ ished fact that a reader will be distracted by the readable content.</p>
+                </li>
+                <li>
+                    <span>
+                        <img src="assets/project.png" alt="ontime"/>
+                    </span>
+                    <h5>Onetime Project</h5>
+                    <p>It is a long establ ished fact that a reader will be distracted by the readable content.</p>
+                </li>
+                <li>
+                    <span>
+                        <img src="assets/design.png" alt="ontime"/>
+                    </span>
+                    <h5>Amazing Design</h5>
+                    <p>It is a long establ ished fact that a reader will be distracted by the readable content.</p>
+                </li>
+                <li>
+                    <span>
+                        <img src="assets/skache.png" alt="ontime"/>
+                    </span>
+                    <h5>Make From Sketch</h5>
+                    <p>It is a long establ ished fact that a reader will be distracted by the readable content.</p>
+                </li>
+            </ul>
+        </div>
+    </section>
+    <section class="pricing">
+    <div class="container">
+       <div class="pricing-head">
+           <h2>Our Services Pricing</h2>
+           <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. </p>
+       </div>
+    <ul class="flex flex-wrap">
+        <li>
+            <h3>HTML Website</h3>
+            <span class="line"></span>
+            <p>
+                Free One Year Domain <br/>
+                Free Installation <br/>
+                Free 1GB Linux Hosting <br/>
+                Unlimited Revision <br/>
+                Five Alternative Designs <br/>
+                Lifetime Support <br/>
```

Comment:
Why are we using `br` tags for line breaks in paragraph ? Is this a good approach ? What is the best practice to do in this case ?

Reply:
I have no use span and p for other lines am just fusing br and move on text next line

Reply:
I dont think so it is a good approach. Please google it and find an other better way to do this.

Reply:
ok sure i will check